cluster-autoscaler/cloudprovider/azure/azure_config.go

```diff
@@ -42,27 +43,9 @@ const (
 
 	imdsServerURL = "http://169.254.169.254"
 
-	// backoff
-	backoffRetriesDefault  = 6
```

Comment:
Delegated back to `cloud-provider-azure`

Reply:
have we confirmed that the cloud-provider-azure defaults are the same as those listed here?

Reply:
Yes I did. 
Also considering about having unit tests to guarantee them in a long run, but may not be necessary if we rely on user to check cloud-provider-azure directly.